test/install.jl

```diff
@@ -5,6 +5,7 @@ import IJulia, JSON
     let kspec = IJulia.installkernel("ijuliatest", "-O3", "-p2",
                     env=Dict("FOO"=>"yes"), specname="Yef1rLr4kXKxq9rbEh3m")
         try
+            @test endswith(kspec, "Yef1rLr4kXKxq9rbEh3m")  # should not contain Julia version suffix
```

Comment:
Should this be testing `==` equality since we want the final name to be strictly equal to `specname`? (modulo debug builds)

Reply:
If we split off the last part `kspec` (which I think is the full path), then yes. Sure, that sounds good!

Reply:
Ah yes sorry, I guess it should be comparing `basename(kspec)`.